src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -172,16 +192,9 @@ class CompanyStep extends React.Component {
                         <ExpensiTextInput
                             label={this.props.translate('common.companyAddress')}
                             containerStyles={[styles.mt4]}
-                            onChangeText={(addressStreet) => {
-                                if (error === this.props.translate('bankAccount.error.addressStreet')) {
-                                    hideBankAccountErrors();
-                                }
-                                this.setState({addressStreet});
-                            }}
+                            onChangeText={getTextChangeHandler('addressStreet', 'companyStepAddressStreet')}
```

Comment:
>if we don't clear the errors immediately when the user is changing the value, it may confuse the user into thinking that it is still wrong

That's a good point. Should we bring this back to the [Slack thread](https://expensify.slack.com/archives/C020EPP9B9A/p1630092331070800) before continuing? 
Maybe @kevinksullivan and @MitchExpensify can advise.

>And I think it will be even better if we just get rid of the error of the field that is being changed, but for that we need the errorKey. What do you think about deriving it consistently from the valueKey

I'm not too sure what the `getErrorKey()` method is doing there. Maybe we can start with something simpler for now?

Reply:
> I'm not too sure what the getErrorKey() method is doing there. Maybe we can start with something simpler for now?

This is just assuming that in the `reimbursementAccount.errors` we will be storing errors like `companyStepPhoneNumber` instead of just `phoneNumber`. If we store them just with the """key""" of the input instead of adding `companyStep` it would make our life easier :)

Reply:
Ok got it! Let's start with the easy thing and see how far we can get :)